File: fizzbuzz.py
# Part 1
def printFizzBuzz(n):
  if n % 3 == 0 and n % 5 == 0:
    print("fizzbuzz")
  elif n % 3 == 0:
    print("fizz")
  elif n % 5 == 0:
    print("buzz")
  else:
    print(n)
    
for i in range(1, 101):
  printFizzBuzz(i)

File: fizzbuzz.js
// Part 2
function printFizzBuzz(n){
    if (n % 5 == 0 && n % 3 == 0){
        console.log("fizzbuzz");
    } else if (n % 5 == 0){
        console.log("buzz");
    } else if (n % 3 == 0){
        console.log("fizz");
    } else{
        console.log(n);
    }
}

for (let i = 1; i < 101; i++) {
    printFizzBuzz(i);
}